AISKU/tsconfig.json

```diff
@@ -9,8 +9,9 @@
     "alwaysStrict": true,
     "suppressImplicitAnyIndexErrors": true,
     "allowSyntheticDefaultImports": true,
-    "importHelpers": false,
+    "importHelpers": true,
     "noEmitHelpers": true,
+    "skipLibCheck": true,
```

Comment:
No, this is so the build actually doesn't fail -- without this TypeScript fails to compile because of breaking changes in tslib 1.13.0 (the reason we introduced the shims module).